packages/compass-collection/package-lock.json

```diff
@@ -1104,7 +1104,7 @@
     },
     "@babel/helper-module-imports": {
       "version": "7.13.12",
-      "resolved": "https://registry.npmjs.org/@babel/helper-module-imports/-/helper-module-imports-7.13.12.tgz",
+      "resolved": "http://localhost:4873/@babel%2fhelper-module-imports/-/helper-module-imports-7.13.12.tgz",
```

Comment:
This looks like it was generated against a locally running registry, we probably don't want to commit that

Reply:
Resolved in e7791b8